terraform/modules/fargate-service/alb.tf

```diff
@@ -0,0 +1,38 @@
+resource "aws_lb" "lb" {
+  name                             = "${var.name_prefix}-lb"
+  internal                         = true
+  load_balancer_type               = "application"
+  subnets                          = var.private_subnets
```

Comment:
Internet-facing, so it should be using the public subnets

Reply:
Have created a bool var for this now - can be set when calling module.